### PR TITLE
BXMSDOC-5183-master: Update installation procedure

### PR DESCRIPTION
**Pull request by csherrar**

### CODE REVIEW

**Comment by domhanak** [comment]
Just tag me on this PR next time. Reviewing here is easier

File: doc-content/enterprise-only/installation/installer-run-proc.adoc [hunk 1]
```diff
@@ -123,7 +123,7 @@ Make a note of the user name and password. You will need them to access {CENTRAL
 . When the installation has completed, click *Next*.
 
 ifdef::PAM[]
-. On the *Configure Runtime Environment* page, choose whether to perform the default installation or perform an advanced configuration.
+. If you installed {KIE_SERVER}, the *Configure Runtime* step appears under *Component Installation*. On the *Configure Runtime Environment* page, choose whether to perform the default installation or perform an advanced configuration.
```
Comment (hmanwani-rh) [inline doc-content/enterprise-only/installation/installer-run-proc.adoc]: I would suggest ... "If {KIE_SERVER} is installed, the *Configure Runtime*...". Also, we can remove whether from the later part of the sentence like "....choose to perform the default installation or perform an advanced configuration."

File: doc-content/enterprise-only/installation/installer-run-proc.adoc [hunk 2]
```diff
@@ -132,12 +132,13 @@ choose to configure database settings or customize certain {KIE_SERVER} options.
...
+. If you selected *Customize {KIE_SERVER} properties*, on the *{KIE_SERVER} Properties Configuration*, change any of the following:
 +
-* Change the name of the {KIE_SERVER} property.
-* Change the URL of the {HEADLESS_CONTROLLER}.
-* Deselect any {KIE_SERVER} server options.
+* *{KIE_SERVER} ID*: The name of the {KIE_SERVER} property.
+ifeval::["{context}" == "install-on-eap"]
+* *Controller URL*: The URL of the {HEADLESS_CONTROLLER}.
+endif::[]
+* Clear any {KIE_SERVER} server function you want to disable.
```
Comment (hmanwani-rh) [inline doc-content/enterprise-only/installation/installer-run-proc.adoc]: I would suggest removing **server**, I believe the **server** word must be included with the attribute, please confirm, if it is not added with the attribute then it's fine. Also, suggest adding **that** like ...."Clear any {KIE_SERVER} function that you want to disable."

**Comment by csherrar** [comment]
> Just tag me on this PR next time. Reviewing here is easier

Will do, thanks @domhanak

**Pull request merged**